index.js

```diff
@@ -222,6 +222,14 @@ module.exports = class BemEntityName {
         return `BemEntityName ${stringRepresentation}`;
     }
     /**
+     * Return raw data for JSON.stringify.
+     *
+     * @returns {*}
+     */
+    toJSON() {
```

Comment:
When I call method `toJSON` I expect to have a json here. This method should be `getData` maybe.